snapcraft.yaml

```diff
@@ -88,18 +104,25 @@ parts:
       - libpng12-0
       - libbz2-1.0
       - libmcrypt4
+      - libfontconfig1
+      - libfreetype6
+      - libjpeg8
     build-packages:
       - libxml2-dev
       - libcurl4-openssl-dev
       - libpng12-dev
       - libbz2-dev
       - libmcrypt-dev
+      - libgd-dev
 
   # Copy over our PHP configuration file.
   php-config:
     plugin: copy
     files:
-      src/php/php.ini: php.ini
+      src/php/php.ini: php/lib/php.ini
+      src/php/extensions.ini: php/lib/php.conf.d/extensions.ini
+      src/php/php-fpm.conf: php/etc/php-fpm.conf
+      src/php/php-fpm: bin/php-fpm
```

Comment:
As of Snapcraft v2.7 you can leave off the destination file names if you don't want to rename them, so this becomes:

```yaml
src/php/php.ini: php/lib/
src/php/extensions.ini: php/lib/php.conf.d/
# ...
```

Less typing, less error-prone.